components/CardUser/CardUser.js

```diff
@@ -0,0 +1,127 @@
+import React, { Fragment } from 'react';
+import Link from 'next/link';
+
+const CardUser = ({title, img, url})=>{
+    return(
+    <Fragment>
+        <Link href={`${url}`}>
+            <div className="card" >  
+                <img src={img} alt={title} className="card-image"/>
+                <h3 className="card-title">{title}</h3>
+                <div className="arrow">
+                    <img src="../static/bilog-arrow-down.svg" />  
+                </div> 
+                
+            </div>
+        </Link>
+
+        <style jsx>
+        {`
+        
+        {/* STYLES FOR MOBILE */}
+        @media only screen and (max-width: 750px){
+            .card{
+                width:15em;
+                height:11em;
+                border-radius: 5px;
+                overflow: hidden;
+                padding:0;
+                margin: 1.5em 0.7em;
+                border-bottom: 4px solid #009EE3;
+                box-shadow: 0 10px 30px rgba(0, 0, 0, 0.12);
+                text-align: center;
+            }
+            
+            .card-image{
+                height:5em;
+                padding:0;
+                margin:0;
+                box-sizing:border-box;
+            }
+            
+            .card-title {
+                height:40px;
+                font-weight: 150;
+                font-size:1em;
+                box-sizing:border-box;
+                padding-top:1.5em;
+                margin:0;
+                color:#333333;
```

Comment:
```suggestion
                color:#333;
```
Cuando los valores hexa se repiten de esa forma, se puede abreviar con los 3 numeros.